package.json

```diff
@@ -53,6 +53,8 @@
   },
   "dependencies": {
     "@workablehr/riviere": "^1.4.0",
+    "cls-hooked": "^4.2.2",
+    "cls-rtracer": "^1.3.0",
```

Comment:
Do we need cls-rtracer? We already handle requestId and this seems like sth easy to implement with cls-hooked only. @klesgidis wdyt?

Reply:
I saw the code that handles the context and the namespaces and I think it's more easier to just use the middleware and let them do the work for us. You can have a look here: https://github.com/puzpuzpuz/cls-rtracer/blob/master/index.js#L47